Solutions/CohesitySecurity/Playbooks/Cohesity_Restore_From_Last_Snapshot/azuredeploy.json

```diff
@@ -0,0 +1,363 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Restore from Last Snapshot",
+        "description": "This playbook allows you to restore your data from a clean snapshot",
+        "prerequisites": "",
+        "postDeployment": [],
+        "prerequisitesDeployTemplateFile": "",
```

Comment:
Not seeing that all Entries related to metadata has been filled, Kindly fill them

Reply:
Done